source/Date.h

```diff
@@ -2,25 +2,37 @@
 #ifndef DATE_H
 #define DATE_H
 
-#include <ctime>
+#include <stdint.h>
```

Comment:
Why are we including this? Is it really necessary?

Reply:
Yes, it is to use uint8_t and uint17_t